specs-go/config.go

```diff
@@ -161,6 +161,8 @@ type Linux struct {
 	// IntelRdt contains Intel Resource Director Technology (RDT) information
 	// for handling resource constraints (e.g., L3 cache) for the container
 	IntelRdt *LinuxIntelRdt `json:"intelRdt,omitempty"`
+	// VTPM configuration
+	VTPMS []*VTPM `json:"vtpms"`
```

Comment:
None of our other arrays are arrays of pointers.  Can we use `[]VTPM` here?

And with aggressive namespacing (#567) we probably want `[]LinuxVTPM` or some such.

Reply:
fixed

Reply:
Also added 'omitempty' now.

Reply:
The runtime-spec and runc differ significantly in this regard. In runc there are arrays with pointers, such as `Mounts []*Mount` and `Devices []*Device`. How do the runtime-spec and runc relate to each other in regards to these files ? If I don't make these changes in runc but leave it at *VTPM, I suppose this will be alright (not to create more churn...)?